app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_cat_age_and_dog_age_equal_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_get_cat_age_and_dog_age_one() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_get_cat_age_and_dog_age_two() -> None:
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_get_cat_age_and_dog_age_equal_different() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_get_cat_age_and_dog_age() -> None:
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
Consider using `@pytest.mark.parametrize` to streamline these tests and avoid repetition. This will make your tests more maintainable and easier to read.